project/src/components/property-card/property-card.tsx

```diff
@@ -55,8 +60,15 @@ export default function PropertyCard(props: PropertyCardProps): JSX.Element {
             <span className="place-card__price-text">/&nbsp;night</span>
           </div>
           <button
-            className="place-card__bookmark-button button"
+            className={`place-card__bookmark-button button ${isFavorite && 'place-card__bookmark-button--active'}`}
             type="button"
+            onClick={() => (
+              dispatch(changeFavoriteStatusAction(
+                {
+                  status: +!isFavorite,
```

Comment:
Рабочий вариант но я предпочитаю более явную запись
```suggestion
                  status: Number(!isFavorite)
```